lib/writer.js

```diff
@@ -4,12 +4,16 @@ var Bluebird = require('bluebird');
 
 var DEFAULT_TYPE = 'other';
 var TYPES = {
+  build: 'Build system / dependencies',
```

Comment:
To match the other types, can you capitalize all of the words? `Build System / Dependencies`